oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConstants.java

```diff
@@ -27,8 +27,9 @@ private OracleConstants() {
   }
 
   public static final String PLUGIN_NAME = "Oracle";
-  public static final String ORACLE_CONNECTION_STRING_SID_FORMAT = "jdbc:oracle:thin:@%s:%s:%s";
-  public static final String ORACLE_CONNECTION_STRING_SERVICE_NAME_FORMAT = "jdbc:oracle:thin:@//%s:%s/%s";
+  // Updating connection strings to accept protocol (e.g., jdbc:oracle:thin:@<protocol>://<host>:<port>/<SID>)
+  public static final String ORACLE_CONNECTION_STRING_SID_FORMAT = "jdbc:oracle:thin:@%s:%s:%s/%s";
+  public static final String ORACLE_CONNECTION_STRING_SERVICE_NAME_FORMAT = "jdbc:oracle:thin:@%s://%s:%s/%s";
```

Comment:
So, `%d` was the problem.

Reply:
No It was actually, '/' We have to use it instead of ':' after port no while specifying the protocol . Removed %d for ports to minimize the change it was %s before.